hermes/file_gen.go

```diff
@@ -1,40 +1,33 @@
 package hermes
 
-
 import (
 	"fmt"
-	"strconv"
 	"crypto/sha1"
 )
 
 type HermesFile struct {
 	name string
 	contents string
 }
-func (file *HermesFile) GenerateFileName(file_ID int, file_checksum string) {
-	id_string := ""
-	if file_ID < 10 {
-		id_string = "0" + strconv.Itoa(file_ID)
-	} else {
-		id_string = strconv.Itoa(file_ID)
-	}
-	file.name := "Hermes_"+id_string+file_checksum
+func (file *HermesFile) generateFileName(file_ID int, file_checksum string) {
+	file_name :=  fmt.Sprintf("Hermes_%02d%v", file_ID, file_checksum)
+	file.name = file_name
```

Comment:
- [ ] There is no need to declare intermediate variables whose sole role is to be assigned to another struct field. Just assign to the struct field directly.

- [ ] Just for a bit of parallelism with the file_id, lets prepend the `_` for the checksum here.

```suggestion
	file.name :=  fmt.Sprintf("Hermes_%02d_%v", file_id, file_checksum)
```